tracing-subscriber/src/filter/env/directive/builder.rs

```diff
@@ -0,0 +1,102 @@
+use core::fmt::Debug;
+
+use tracing_core::LevelFilter;
+
+use crate::filter::{env::field, ValueMatch};
+
+use super::Directive;
+
+/// A [builder] for constructing new [`Directive`]s.
+///
+/// [builder]: https://rust-unofficial.github.io/patterns/patterns/creational/builder.html
+#[derive(Debug, Clone)]
+#[must_use]
+pub struct Builder {
+    in_span: Option<String>,
+    fields: Vec<field::Match>,
+    target: Option<String>,
+    level: LevelFilter,
+}
+
+// ==== impl Builder ====
+
+impl Builder {
```

Comment:
:exclamation:  Currently we could add all of this methods directly onto `Directive` but as idk. if that will be the case in the future I added a builder.